City-of-Helsinki/berth-reservations-admin
# VEN-696 | Render harbors list data when switching from the single harbor page

## Description :sparkles:
- Add pier's id to the harbors query

### Closes :no_good_woman:
[VEN-696](https://helsinkisolutionoffice.atlassian.net/browse/VEN-696): Harbors list data is not rendered when switching from single harbor page

### Manual testing :construction_worker_man:
- Land straight to single harbor view
- Click “Venepaikat” from the left menu
- The list should render the harbors properly
